lib/emitter/src/Events/CourseModuleCompleted.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
```

Comment:
As much as I want you to be credited with this work, I have a number of issues with these comments:
1. Github offers a number of more accurate functions for this (blame, history, contributor graphs, etc).
2. Likely not to be maintained, hence incorrectly giving you credit/blame for things in this file.